com.github.najepaliya.kleaner.json

```diff
@@ -0,0 +1,110 @@
+{
+    "id": "com.github.najepaliya.kleaner",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15-22.08",
+    "sdk": "org.kde.Sdk",
+    "command": "kleaner",
+    "finish-args": [
+        "--device=dri",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--filesystem=host"
+    ],
+    "modules": [
```

Comment:
probably missing cleanup rules to remove build only file like includes and pkgconfig files, and also .a

Reply:
Yes, will fix.